rclpy/topics/minimal_publisher/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>examples_rclpy_minimal_publisher</name>
+  <version>0.0.0</version>
+  <description>Examples of minimal publishers using rclpy.</description>
+
+  <maintainer email="mikael@osrfoundation.org">Mikael Arguedas</maintainer>
+  <license>Apache License 2.0</license>
+
+  <exec_depend>rclpy</exec_depend>
+  <exec_depend>std_msgs</exec_depend>
+
+  <test_depend>ament_copyright</test_depend>
+  <test_depend>ament_pep257</test_depend>
+  <test_depend>ament_pep8</test_depend>
+  <test_depend>ament_pyflakes</test_depend>
```

Comment:
I thought the conclusion was exactly the opposite, because testing would cause the CMake and potentially the code to be more complex, making it less copy-pastable. The tutorials, however, are not meant as minimal starting locations, so they can afford to have more complex CMake lists and code?